tools/biolatency_example.txt

```diff
@@ -292,11 +292,62 @@ flags = Priority-Metadata-Write
 These can be handled differently by the storage device, and this mode lets us
 examine their performance in isolation.
 
+The -j option prints a dictionary of the histogram.
+In most cases, this flag will be used with the -f flag. For example:
```

Comment:
Please update examples properly here. -f is not available any more.

Reply:
Updated.